mathesar/tests/api/test_table_api.py

```diff
@@ -1025,6 +1025,14 @@ def _get_patents_column_data():
     }]
 
 
+def get_column_data_with_id(table, column_data):
```

Comment:
This fixture can also be converted to accept `table_oid` and return column data  with id